admin-approve.php

```diff
@@ -1,4 +1,26 @@
 <?php
+////The MIT License (MIT)
```

Comment:
It would be better if the Copyright is at the start of the file, in the first line, before PHP